recipes/sip/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [ `uname` == Darwin ]; then
+    pythonw configure.py
+else
+    python configure.py
```

Comment:
In the past, we specified the site-packages and the sip header install locations in an older build system used on campus, but they may already be deduced through other means. As you say, `--sysroot` is probably sufficient and it is pretty easy to get the rest of the stuff out of the interpreter itself.